docs/api/navigators/DrawerNavigator.md

```diff
@@ -157,25 +153,22 @@ Usually you define static `navigationOptions` on your screen component. For exam
 ```jsx
 class ProfileScreen extends React.Component {
 
-  static navigationOptions = {
+  static navigationOptions = ({ navigation }) => ({
 
-    title: ({ state }) => `${state.params.name}'s Profile!`,
+    title: `${navigation.state.params.name}'s Profile!`,
 
-    drawer: {
-      icon: (
-        <Image src={require('./my-icon.png')} />
-      ),
-    },
-  };
+    drawerIcon: (
+      <Image src={require('./my-icon.png')} />
+    ),
+  });
   ...
 ```
 
 All `navigationOptions` for the `DrawerNavigator`:
```

Comment:
I think all options specific to given navigator belong to its `api` documentation (as on this page). I have removed example from that page and added links from `Screen-Nav-Options`. I think that's a good first try.